backend/web/workflows/sft_fixer.py

```
@@ -0,0 +1,969 @@
+            message=f"Instance {validation_request.instance_id} is {'valid and reachable' if valid else 'not reachable'}",
+        )
+
+    except NotImplementedError as e:
+        return ValidateInstanceResponse(
+            valid=False,
+            instance_id=validation_request.instance_id,
+            status="not_implemented",
+            agent_status=None,
+            message=f"Instance validation not yet implemented for this provider: {str(e)}",
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)